src/bfx_tools/gatk.ml

```diff
@@ -639,3 +647,95 @@ let mutect2
     in
     let final_vcf = result_prefix ^ "-merged.vcf" in
     Vcftools.vcf_concat ~run_with targets ~final_vcf ~more_edges
+
+
+(** Combine variant records from different sources.
+
+    cf. https://www.broadinstitute.org/gatk/gatkdocs/org_broadinstitute_gatk_tools_walkers_variantutils_CombineVariants.php  *)
+let combine_variants
+    ?(configuration=Configuration.Combine_variants.default)
+    ?(edges=[])
+    ~run_with
+    ~variant_sources
+    ~output_vcf_path
+    ~genotype_merge_option
+  =
+  let open KEDSL in
+  let reference =
+    Machine.get_reference_genome run_with "" in (* how to get the reference we want to use? *)
```

Comment:
pass a `~reference_genome` argument?
Like here: https://github.com/hammerlab/biokepi/blob/c2aac7c1702375485268ae2c1c74f5d9d11a53ff/src/pipeline_edsl/semantics.ml#L188


Later, when this one and #268 are merged, we can create a proper `vcf` workflow-product that (like we do in `bam_file`) carries around its reference-genome.